src/components/Articles.jsx

```diff
@@ -1,9 +1,38 @@
-function Articles(){
-    return (
-        <h3>articles hello</h3>
+import { useEffect,useState } from "react"
+
 
 
-    )
+function Articles(){
+    const [articlesArr, setArticlesArr] = useState([])
+    useEffect(()=>{
+        fetch("https://northcoders-news-api-bjpy.onrender.com/api/articles").then((response)=>{
+            return response.json()
+        }).then(({articles})=>{
+            setArticlesArr(articles)
+        })
+    },[])
+
+    if(articlesArr.length){
```

Comment:
we dont need the length check on the map as if the array is empty, it wont return anything at all! :)